llvm/include/llvm/CAS/FileSystemCache.h

```diff
@@ -43,14 +43,15 @@ class FileSystemCache : public ThreadSafeRefCountedBase<FileSystemCache> {
   struct DirectoryListingInfo;
 
   struct LookupPathState {
+    FileSystemCache *Cache;
```

Comment:
Is all we need here is `PathStyle`? If so, I will avoid pulling in the entire FileSystemCache in case that was mis-used.

Reply:
Done.